src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+public class Task {
+    private final String description;
+    private boolean isDone;
+
+    public static final String TICK = "[\u2713]";
+    public static final String CROSS = "[\u2718]";
+
+    public Task (String description) {
+        this.description = description;
+        isDone = false;
+    }
+
+    public void setIsDone(Boolean bool) {
+        this.isDone = bool;
```

Comment:
Should consider a more descriptive name here instead of bool. Since you are using the keyword "this", the name can be isDone also.